MHVLib/Models/Categories/NSString+Utils.m

```diff
@@ -0,0 +1,415 @@
+//
+// NSString+Utils.m
```

Comment:
Most of these category methods are not used and are very specific to the HVI app. It looks like the only methods being used are '**boolString**' and '**capitalizedStringForSelectors**'. Let's move these 2 methods into MHVStringExtensions, so string extensions are all located in one spot.